### ISSUE TITLE
Loss not decreasing

### ISSUE THREAD

**Issue by HanbinJeon** [hmorimitsu/ptlflow#98]
I tried to train pwcnet using
--model pwcnet --data.train_dataset chairs-val --data.val_dataset chairs-val 
but the loss doesn't decrease in the tensorboard and keeps shaking.
I tried RAFT but it's the same too.
Why is this happening?
